File: src/Order/Domain/Entities/OrderItem.ts
export class OrderItem {
    id: string;
    product_uuid: string;
    quantity: number;
    price: number;
    category: string;

    constructor(
        id: string,
        product_uuid: string,
        quantity: number,
        price: number,
        category: string
    ) {
        this.id = id;
        this.product_uuid = product_uuid;
        this.quantity = quantity;
        this.price = price;
        this.category = category;
    }
}

File: dashboard/Domain/entities/order_item.py
class OrderItem:
    def __init__(self, orderItemUuid, product_uuid, quantity, price, category, orderOrderUuid):
        self.orderItemUuid = orderItemUuid
        self.product_uuid = product_uuid
        self.quantity = quantity
        self.price = price
        self.category = category
        self.orderOrderUuid = orderOrderUuid
